src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -389,18 +390,26 @@ struct uct_ib_mlx5_cmd_hca_cap_bits {
 
     uint8_t    reserved_at_500[0x20];
     uint8_t    num_of_uars_per_page[0x20];
-    uint8_t    reserved_at_540[0x40];
+
+    uint8_t    flex_parser_protocols[0x20];
+
+    uint8_t    max_geneve_tlv_options[0x8];
+    uint8_t    reserved_at_568[0x3];
+    uint8_t    max_geneve_tlv_option_data_len[0x5];
+    uint8_t    reserved_at_570[0x10];
 
     uint8_t    reserved_at_580[0xb];
     uint8_t    log_max_dci_stream_channels[0x5];
     uint8_t    reserved_at_590[0x3];
     uint8_t    log_max_dci_errored_streams[0x5];
-    uint8_t    reserved_at_598[0x8];
+    uint8_t    log_max_atomic_size_dc[0x8];
 
-    uint8_t    reserved_at_5a0[0x10];
+    uint8_t    max_multi_user_group_size[0x10];
```

Comment:
where is it used?

Reply:
I aligned our bits with driver

Reply:
pls leave just fields required for this PR

Reply:
done